api/graphql/mutations/role.js

```diff
@@ -0,0 +1,88 @@
+const { GraphQLYogaError } = require('@graphql-yoga/node')
+
+export async function addGroupRole ({ groupId, color, name, emoji, userId }){
+  if (!userId) throw new GraphQLYogaError(`No userId passed into function`)
+
+  if (groupId && name && emoji) {
+    const groupMembership = await GroupMembership.forIds(userId, groupId).fetch()
+
+    if (groupMembership && (groupMembership.get('role') === GroupMembership.Role.MODERATOR)) {
+      const newGroupRole = { group_id: groupId, name, emoji, active: true }
+      if (color) newGroupRole.color = color
```

Comment:
I think you can just include color in the object above and if its null it will be ignored